apps/condo/domains/banking/schema/BankCategory.test.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * Generated by `createschema banking.BankCategory 'name:Text;'`
+ */
+
+const { makeLoggedInAdminClient, makeClient, UUID_RE, DATETIME_RE, expectToThrowGQLError } = require('@open-condo/keystone/test.utils')
+
+const {
+    expectToThrowAuthenticationErrorToObj, expectToThrowAuthenticationErrorToObjects,
+    expectToThrowAccessDeniedErrorToObj,
+} = require('@open-condo/keystone/test.utils')
+
+const { makeClientWithNewRegisteredAndLoggedInUser } = require('@condo/domains/user/utils/testSchema')
+
+const { BankCategory, createTestBankCategory, updateTestBankCategory } = require('@condo/domains/banking/utils/testSchema')
+
+describe('BankCategory', () => {
+    describe('CRUD tests', () => {
+        describe('create', () => {
+            test('admin can', async () => {
+                const admin = await makeLoggedInAdminClient()
```

Comment:
Please, move actors initialisation to before all... 65mins test job is not funny at all :(